Only in aaplus_1_59: aaplus_wrap.cxx
diff -ru aaplus_1_59_orig/AASaturn.h aaplus_1_59/AASaturn.h
--- aaplus_1_59_orig/AASaturn.h	2014-11-12 00:22:30.000000000 +0100
+++ aaplus_1_59/AASaturn.h	2015-02-27 21:41:32.691243019 +0100
@@ -42,8 +42,8 @@
   static double EclipticLatitude(double JD);
   static double RadiusVector(double JD);
 
-  static double EclipticLongitudeJ2000(double JD);
-  static double EclipticLatitudeJ2000(double JD);
+//  static double EclipticLongitudeJ2000(double JD);
+ // static double EclipticLatitudeJ2000(double JD);
 };
 
 #endif //__AASATURN_H__
